internal/interactions/gitlabintr.go

```diff
@@ -17,14 +17,14 @@ type GitlabIntr interface {
 	ListGroups(opt *gitlab.ListGroupsOptions) ([]*gitlab.Group, *gitlab.Response, error)
 	GetProject(pid interface{}) (*gitlab.Project, *gitlab.Response, error)
 	GetNamespace(id interface{}) (*gitlab.Namespace, error)
-	CreateProject(opt *gitlab.CreateProjectOptions) error
+	CreateProject(opt *gitlab.CreateProjectOptions) (*gitlab.Project, error)
```

Comment:
As the interface changes here this looks like a backward incompatible change.

Reply:
this one is in internal and should't impact any user